### PR TITLE
Add v2.1

### PR DESCRIPTION
**Pull request by lpichler**
based on request in https://github.com/RedHatInsights/topological_inventory-api/pull/284#discussion_r384003101

cc @bdunne @abellotti

### CODE REVIEW

File: app/controllers/api/v2x1.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+module Api
+  module V2x1
+    class RootController < ApplicationController
+      def openapi
+        render :json => ::Insights::API::Common::OpenApi::Docs.instance["2.1"].to_json
+      end
+    end
+
+    # Same behavior as api/v1.0/
```
Comment (bdunne) [inline app/controllers/api/v2x1.rb]: These should inherit from v2.0, not v1.0

Comment (lpichler) [reply app/controllers/api/v2x1.rb]: done 👍

Comment (bdunne) [reply app/controllers/api/v2x1.rb]: The comment is incorrect now.  It can be removed though.

Comment (lpichler) [reply app/controllers/api/v2x1.rb]: yes, removed, thanks 👍

**Pull request closed without merging**